files/es/glossary/python/index.html

```diff
@@ -20,5 +20,5 @@ <h2 id="Aprende_Más">Aprende Más</h2>
 
 <ul>
  <li>{{interwiki('wikipedia','Python','Python')}} en Wikipedia</li>
- <li><a href="http://docs.python.org.ar/tutorial/3/index.html">Tutorial de Python en Español</a></li>
+ <li><a href="https://tutorial.python.org.ar/en/latest/">Tutorial de Python en Español</a></li>
```

Comment:
Hola, @epolicardo, muchas gracias por tu aportación!

Gracias por el interes en corregir el tutorial de Python, fuí [a ver la documentación](https://developer.mozilla.org/en-US/docs/Glossary/Python) en ingles, ahí estan utilizando esta documentación de python https://docs.python.org/3/tutorial/index.html, por lo tanto la documentación en español que debemos utilizar se encuentra en esta dirección https://docs.python.org/es/3/tutorial/index.html.

podrias actualizarlo, por favor?



Más acerca de MDN en Español:
* [Actualizar el contenido con la versión en inglés](https://github.com/mdn/translated-content/discussions/4029)
* [Revisar issues](https://github.com/mdn/translated-content/issues?q=is%3Aopen+is%3Aissue+label%3Al10n-es)
* [Unirte a nuestro equipo](https://chat.mozilla.org/#/room/!cIEBGoIHFpsXNeVUwp:mozilla.org)

Saludos.

Reply:
If I may, a Spanish version of the tutorial is available at https://docs.python.org/es/3/tutorial/index.html (feel free to link to this instead of the English one since this document is for the Spanish locale :))

Reply:
> If I may, a Spanish version of the tutorial is available at https://docs.python.org/es/3/tutorial/index.html (feel free to link to this instead of the English one since this document is for the Spanish locale :))

thank you, it is the one I suggested using!, you are the best :+1:

Reply:
Ah my bad, I didn't see that (I thought it was made as a suggestion, should have paid attention 😅)

Reply:
> Ah my bad, I didn't see that (I thought it was made as a suggestion, should have paid attention sweat_smile)

no worries! Thanks!